tracer/src/Datadog.Trace/ClrProfiler/NativeCallTargetDefinition.cs

```diff
@@ -24,6 +24,13 @@ internal struct NativeCallTargetDefinition
 
         public ushort TargetSignatureTypesLength;
 
+        [MarshalAs(UnmanagedType.U1)]
```

Comment:
`bool` vs `BOOL` :)

Reply:
Is this fine or do I need to make a change?

Reply:
Perfectly fine!, you don't need to modify anything. I was just pointing out this line is required because you used `bool` instead of `BOOL` in the native struct.